packages/dhis2/README.md

```diff
@@ -1,98 +1,107 @@
 # Language DHIS2
 
-An OpenFn language Pack for building expressions and operations for working with
-the
-[DHIS2 API](http://dhis2.github.io/dhis2-docs/master/en/developer/html/dhis2_developer_manual.html).
-Used most commonly via OpenFn.org or by hand with
-[OpenFn/core](https://github.com/OpenFn/core) from the root of this repo:
+An OpenFn language pack for building expressions and operations to work with the
+[DHIS2 API](https://docs.dhis2.org/en/home.html). Commonly used via
+[OpenFn.org](https://app.openfn.org) or manually with
+[OpenFn CLI](https://github.com/OpenFn/kit).
+
+## Table of Contents
+
+1. [Getting Started](#getting-started)
+2. [Important Note](#important-note)
+3. [Configuration](#configuration)
+4. [Helper Functions](#helper-functions)
+5. [Development](#development)
+   - [Running Tests](#running-tests)
+   - [Unit Tests](#unit-tests)
+   - [Integration Tests](#integration-tests)
+   - [Troubleshooting Tests](#troubleshooting-tests)
+
+---
+
+## Getting Started
+
+To use this package, execute commands via OpenFn/core from the root of the
+repository:
 
 ```sh
-core execute \
- -l ../language-dhis2 \
- -e ./tmp/expression.js \
- -s ./tmp/state.json \
- -o ./tmp/output.json
+openfn job.js -a dhis2 -s tmp/state.json -o tmp/output.json
```

Comment:
The CLI will write to `tmp/output.json` by default so I would exclude it from the command  - keep it shorter and neater

Reply:
Updated the command